pkgs/by-name/de/deltatouch/package.nix

```diff
@@ -0,0 +1,93 @@
+{ lib
+, stdenv
+, fetchFromGitea
+, cmake
+, intltool
+, libdeltachat
+, lomiri
+, qt5
+, quirc
+}:
+
+stdenv.mkDerivation {
+  pname = "deltatouch";
+  version = "1.3.1-unstable-2024-02-25";
+
+  src = fetchFromGitea {
+    domain = "codeberg.org";
+    owner = "lk108";
+    repo = "deltatouch";
+    rev = "dd86a77291a6e9f1c363e7cdf5999e58fb5b84e5";
+    hash = "sha256-03J2YLXF1uDnTmDsrxWjlfDsN7mcc68MbWy/HTIcM2E=";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [
+    qt5.wrapQtAppsHook
+    intltool
+    cmake
+  ];
+
+  buildInputs = [
+    qt5.qtbase
+    qt5.qtwebengine
+    qt5.qtquickcontrols2
+    lomiri.lomiri-ui-toolkit
+    lomiri.lomiri-ui-extras
+    lomiri.lomiri-api
+    lomiri.lomiri-indicator-network # Lomiri.Connectivity module
```

Comment:
Readme of the Nixpkgs branch / that commit says:

> […] Differences to the Ubuntu Touch version are:
> - […]
> - The Lomiri Connectivity module is not used

Is this needed here, is the readme outdated?

Reply:
I'm sorry, the readme is outdated, will fix asap

Reply:
Should we switch this to the main branch then?

Reply:
Yes. I had not informed link2xt about this because I did not have time before to test whether it runs from main. I just checked, it does.